README.md

```diff
@@ -283,3 +283,10 @@ X. Exit testing.
 Any option can also be applied to all future occurences of that type of failure
 by adding `!` after it. For example, if you want to mark *all* failing specs as
 `:todo` for the current implementation you'd type `I!`.
+
+## Tests
+The unit tests are located in the `tests/` directory. To run the tests, run
```

Comment:
It's probably a good idea to be clear here that these are unit tests *for the spec runner itself*, rather than tests for the Sass language.